lms/templates/instructor/instructor_dashboard_2/keyword_sub_modal.html

```diff
@@ -0,0 +1,47 @@
+<%! from django.utils.translation import ugettext as _ %>
+
+<section id="keyword-sub-modal" class="modal keyword-sub-modal" aria-hidden="true">
+  <div id="keyword-sub-wrapper" class="inner-wrapper" role="dialog" aria-labelledby="keyword-sub-tip-title">
+    <button class="close-modal">
+      <i class="icon-remove"></i>
+      <span class="sr">
+        ## Translators: this is a control to allow users to exit out of this modal interface (a menu or piece of UI that takes the full focus of the screen)
+        ${_("Close")}
+      </span>
+    </button>
+
+    <header>
+      <h2 id="keyword-sub-tip-title">
+        ${_("Keyword Substitution")}
+        <span class="sr">,
+          ## Translators: this text gives status on if the modal interface (a menu or piece of UI that takes the full focus of the screen) is open or not
+          ${_("window open")}
```

Comment:
Text for screen readers to to alert users that they are in an open modal window.